src/vbaDeveloper.xlam/Build.bas

```diff
@@ -147,13 +147,30 @@ Public Sub exportVbaCode(vbaProject As VBProject)
 End Sub
 
 
+Private Function codeModuleExists(component As VBComponent) As Boolean
+    On Error GoTo doesnt
+    Dim tempObj
+    Set tempObj = component.codeModule
+    codeModuleExists = True
+    Exit Function
+doesnt:
+    On Error GoTo -1
+    Err.Clear
+    codeModuleExists = False
+    Exit Function

```

Comment:
```suggestion
    codeModuleExists = False
```

Reply:
Keep it simple. The error is already reset once it's caught by the 'doesnt' error handler.